### ISSUE TITLE
/dist/healthcheck.js not running in kubernetes Context

### ISSUE THREAD

**Issue by jheyduk** [intershop/intershop-pwa#968]
**Actual Behavior**

When configuring readinessProbe in k8s with `node /dist/healthcheck.js` the check fails with following error:
```
  Warning  Unhealthy  4s (x2 over 64s)  kubelet                                Readiness probe failed: internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module './ecosystem-ports'
Require stack:
- /dist/healthcheck.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/dist/healthcheck.js:24:13)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/dist/healthcheck.js' ]
```

**Expected Behavior**

The script should work.

**Steps to Reproduce the Bug**

1. Execute script on your container on k8s using `kubectl exec -ti <your pod> -c manager -- node /dist/healthcheck.js` or on your local machine `docker exec -ti <your container> node /dist/healthcheck.js`
2. See error

**Environment Details**

PWA-Version: 1.4.0

**Additional Context, like Screenshots, Log File Snippets etc.**

[AB#72363](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/72363)

**Comment by Eisie96** [comment]
Please notice, that scripts for the readiness and liveness probe are available in the dist folder since 1.0. These should be used,

**Comment by jheyduk** [comment]
Hi @Eisie96 ,
Thank you for clarification!
